delphi-frontend/src/test/java/au/com/integradev/delphi/compiler/PredefinedConditionalsTest.java

```diff
@@ -368,4 +371,38 @@ void testLinuxOnlySupportsNextGenFeaturesBeforeDelphiRio() {
     assertThat(PredefinedConditionals.getConditionalDefines(Toolchain.DCCLINUX64, VERSION_RIO))
         .doesNotContainAnyElementsOf(NEXT_GEN_FEATURES);
   }
+
+  @ParameterizedTest
+  @EnumSource(
+      value = Toolchain.class,
+      names = {
+        "DCCOSX64",
+        "DCCOSXARM64",
+        "DCCIOSARM",
+        "DCCIOSARM64",
+        "DCCIOSSIMARM64",
+        "DCCAARM",
+        "DCCAARM64",
+        "DCCLINUX64"
+      })
+  void testLLVMToolchainsAfterDelphiAlexandriaShouldDefineLLVM(Toolchain toolchain) {
+    assertThat(PredefinedConditionals.getConditionalDefines(toolchain, VERSION_ATHENS))
+        .contains("LLVM");
+  }
+
+  @ParameterizedTest
+  @EnumSource(
+      value = Toolchain.class,
+      names = {"DCC32", "DCC64", "DCCOSX", "DCCIOS32"})
+  void testNonLLVMToolchainsAfterDelphiAlexandriaShouldDefineLLVM(Toolchain toolchain) {
```

Comment:
I think this should be "should**Not**DefineLLVM".

Reply:
Oops. 👀